_posts/2020-06-12-solidity-0.6-inheritance.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+published: true
+title:  "Solidity 0.6.x features: inheritance"
+date:   2020-06-12
```

Comment:
Should the date be updated?

Reply:
Sure. Done

Reply:
Sorry, make sure to also rename the file to match the date